ansible/roles/debops.docker/defaults/main.yml

```diff
@@ -0,0 +1,391 @@
+                       if (ansible_local|d() and ansible_local.pki|d() and
+                           ansible_local.pki.realm|d())
+                       else "domain" }}'
+
+                                                                   # ]]]
+# .. envvar:: docker__pki_ca [[[
+#
+# Default root CA certificate used by the Docker client, relative to
+# :envvar:`docker__pki_realm`.
+docker__pki_ca: 'CA.crt'
+
+                                                                   # ]]]
+# .. envvar:: docker__pki_crt [[[
+#
+# Default certificate used by the Docker client, relative to
+# :envvar:`docker__pki_realm`.
+docker__pki_crt: 'default.crt'
+
+                                                                   # ]]]
+# .. envvar:: docker__pki_key [[[
+#
+# Default private key used by the Docker client, relative to
+# :envvar:`docker__pki_realm`.
+docker__pki_key: 'default.key'
+
+                                                                   # ]]]
+# .. envvar:: docker__tls_verify [[[
+#
+# Whether the Docker client should verify the authenticity of the Docker host
+# by default.
+docker__tls_verify: '{{ docker__pki }}'
+
+                                                                   # ]]]
+# .. envvar:: docker__tls_hostname [[[
+#
+# The default server name used to verify the authenticity of the Docker host.
+docker__tls_hostname: '{{ ""
+                          if (ansible_local|d() and
+                              ansible_local.docker_server|d() and
+                              ansible_local.docker_server.installed|d())
+                          else "docker." + ansible_domain }}'
+
+                                                                   # ]]]
+# .. envvar:: docker__ssl_version [[[
+#
+# The default ssl.py SSL/TLS version used to connect to the Docker host.
+docker__ssl_version: 'TLSv1_2'
+                                                                   # ]]]
+                                                                   # ]]]
+# Default container configuration [[[
+# -----------------------------------
+
+# .. envvar:: docker__auto_remove [[[
+#
+# If True, enable auto-removal of containers by default. This will remove a
+# container as soon as the container's process exits.
+docker__auto_remove: False
+
+                                                                   # ]]]
+# .. envvar:: docker__debug [[[
+#
+# Enable or disable debug mode.
+docker__debug: False
+
+                                                                   # ]]]
+# .. envvar:: docker__pull [[[
+#
+# If True, always pull the latest version of an image by default. Otherwise,
+# will only pull an image when missing.
+docker__pull: False
+
+                                                                   # ]]]
+# .. envvar:: docker__restart_policy [[[
+#
+# The default container restart policy. Supported values:
+#
+# - no
+# - on-failure
+# - always
+# - unless-stopped
+docker__restart_policy: 'no'
```

Comment:
DebOps is aimed at production environments. I'm not sure what is the convention for Docker in production, shouldn't failed containers always restart automatically? Perhaps a default of `unless-stopped` would work better in this case?

Reply:
Well we only use `unless-stopped` in production, but `no` is the Docker default (which I just copied). I'll change it to `unless-stopped`.